### PR TITLE
Refactor Dockerfile

### PR DESCRIPTION
**Pull request by Gauvino**

### CODE REVIEW

**Review by Shadowghost — APPROVED** [review]

File: deployment/Dockerfile.debian [hunk 1]
```diff
@@ -1,6 +1,4 @@
-ARG IMG=debian
-ARG TAG=buster
-FROM $IMG:$TAG
+FROM debian:bookworm-slim
```
Comment (iwalton3) [inline deployment/Dockerfile.debian:1]: This breaks CI builds for different Debian versions.

File: deployment/Dockerfile.debian [hunk 2]
```diff
@@ -14,9 +12,9 @@ ENV IS_DOCKER=YES
 
 # Prepare Debian build environment
 RUN apt-get update \
-  && DEBIAN_FRONTEND="noninteractive" TZ="America/New_York" \
-     apt-get install -y debhelper mmv git curl devscripts equivs
-
+ && DEBIAN_FRONTEND="noninteractive" TZ="America/New_York" \
+    apt-get install --no-install-recommends -y \
+    debhelper mmv git curl devscripts equivs
```
Comment (iwalton3) [inline deployment/Dockerfile.debian:17]: This looks fine to me.

**Review by iwalton3 — CHANGES_REQUESTED** [review]

File: deployment/Dockerfile.debian (hunk 1 above)
Comment (Gauvino) [reply deployment/Dockerfile.debian:1]: Should i revert this and only changed the tag to bookworm-slim ?

Comment (iwalton3) [reply deployment/Dockerfile.debian:1]: You would want to adjust the CI pipeline
you can also adjust the default options but the dockerfile must still accept the image and tag as parameters

Comment (Gauvino) [reply deployment/Dockerfile.debian:1]: Changing this ?
https://github.com/jellyfin/jellyfin-media-player/blob/ee0717ea933f9bef25cdac6e12d873281d14cc00/.github/workflows/main.yml#L153

**Review by iwalton3 — APPROVED** [review]

**Pull request merged**